Launcher/config.json

```diff
@@ -145,6 +145,11 @@
   // To get your access token go to https://polygon.io
   "polygon-api-key": "",
 
+  // zerodha configuration goto https://kite.trade
+  "zerodha-access-token": "",
+  "zerodha-api-key": "",
+  "zerodha-api-secret": "",
```

Comment:
Is `"zerodha-api-secret"` needed? 
Seems the `ZerodhaBrokerageFactory` only uses the first two settings